app.css

```diff
@@ -0,0 +1,54 @@
+* {
+  box-sizing: border-box;
+  font-size: 10px;
+  font-family: sans-serif;
+  font-weight: bold;
+}
+
+body {
+  width: 900px;
+  margin: auto;
+  color: lightgray;
+}
+
+.cat-column {
+  float: left;
+  display: inline;
+  margin: auto;
+  width: 280;
+  padding: 10px;
+}
+
+.octocats {
+  float: left;
+  width: 275px;
+  padding: 25px;
```

Comment:
Remember `padding` is for space within an element, margin gives room between this and then next element.